packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -638,8 +652,10 @@ export const connectEpic: Epic<AnyAction, AnyAction, { ws: RootState }, any> = (
           action$.pipe(
             // TODO: not seeing unsubscribe events because of this
             filter(disconnectFulfilled.match),
-            tap((action) => logger.info('WS: Disconnected', connectionMeta(action.payload))),
             filter((a) => a.payload.config.connectionInfo.key === connectionKey),
+            tap((action) => {
+              logger.info('WS: Disconnected Fulfilled', connectionMeta(action.payload))
```

Comment:
```suggestion
              logger.debug('WS: Disconnected Fulfilled', connectionMeta(action.payload))
```

Reply:
I think `debug` would be better as there are times when we actually want to disconnect from a WS endpoint.